### PR TITLE
Aws foundational/waf wafv2

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

File: queries/wafv2/wafv2_web_acl_logging_should_be_enabled.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+-- WAF Classic
+SELECT account_id,
+       arn
+FROM aws_waf_web_acls
+WHERE logging_configuration IS NULL
+   OR logging_configuration = '{}';
```
Comment (disq) [inline queries/wafv2/wafv2_web_acl_logging_should_be_enabled.sql:6]:
```suggestion
   OR logging_configuration = '{}'
```

**Review by disq — CHANGES_REQUESTED** [review]

File: queries/wafv2/wafv2_web_acl_logging_should_be_enabled.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+UNION
+-- WAF V2
+SELECT account_id,
+       arn
+FROM aws_wafv2_web_acls
+WHERE logging_configuration IS NULL
+   OR logging_configuration = '{}';
```
Comment (bbernays) [inline queries/wafv2/wafv2_web_acl_logging_should_be_enabled.sql:14]: What happens if logging is configured and then disabled? I know other services don't set the entire object to null/ empty, they just set the fields in the object to `NULL` or empty

**Review by bbernays — CHANGES_REQUESTED** [review]

**Review by disq — APPROVED** [review]
@zagronitay is there a manual test case about what @bbernays described? Can you check if configuring and then disabling logging results in an empty logging configuration, or would it create a `"some_key_disabled":true` (or the opposite) entry?

**Review by disq — APPROVED** [review]

**Review by bbernays — APPROVED** [review]
LGTM

**Pull request merged**